src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6181,6 +6164,22 @@ private BoundExpression BindPointerElementAccess(ExpressionSyntax node, BoundExp
             return new BoundPointerElementAccess(node, expr, index, CheckOverflowAtRuntime, pointedAtType, hasErrors);
         }
 
+        private bool ReportRefOrOutArgument(AnalyzedArguments analyzedArguments, DiagnosticBag diagnostics)
```

Comment:
yes

---
In reply to: [75064780](https://github.com/dotnet/roslyn/pull/13091#discussion_r75064780) [](ancestors = 75064780)